common/src/configuration/utils.rs

```diff
@@ -281,45 +287,64 @@ fn set_mining_node_defaults(cfg: &mut Config) {
     cfg.set_default("mining_node.validate_tip_timeout_sec", 0).unwrap();
 }
 
-fn set_transport_defaults(cfg: &mut Config) {
-    // Mainnet
-    // Default transport for mainnet is tcp
-    cfg.set_default("base_node.mainnet.transport", "tcp").unwrap();
-    cfg.set_default("base_node.mainnet.tcp_listener_address", "/ip4/0.0.0.0/tcp/18089")
-        .unwrap();
+fn set_transport_defaults(cfg: &mut Config) -> Result<(), config::ConfigError> {
+    let apps = &[ApplicationType::BaseNode, ApplicationType::ConsoleWallet];
+    for app in apps {
+        let app = app.as_config_str();
+        // Mainnet
+        cfg.set_default(&format!("{}.mainnet.transport", app), "tor")?;
+        cfg.set_default(
+            &format!("{}.mainnet.tcp_listener_address", app),
+            "/ip4/0.0.0.0/tcp/18089",
```

Comment:
Shouldn't the wallet and base node have different listening ports when using TCP?

Reply:
Ah good catch, I'll fix that